modules/test/playwright/pages/FDS/FDSActions.page.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import type {Page} from '@playwright/test';
+
+import {FDSViewPage} from './FDSView.page';
+
+export type ModalTypes = 'modal' | 'link' | 'sidePanel';
```

Comment:
could we import types from DXP modules? not urgent but I see a duplication [here](https://github.com/liferay/liferay-portal/blob/d777698203a6e37b3882b84fb559a3c3bc6fd315/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx#L43-L49), and no good reason to keep those separated, now that everything is JS/TS.

@izaera I did not realize about this, what do you think?

Reply:
I think it's a good question :sweat_smile: 

This is somewhat related to this discussion -> https://liferay.slack.com/archives/C069U77143Z/p1704821039771309?thread_ts=1703131754.446429&cid=C069U77143Z

Whether we treat tests as something totally decoupled from `liferay-portal` or something that coexists with production code...

I see pros and cons for both scenarios.

Reply:
@izaera The [link](https://liferay.slack.com/archives/C069U77143Z/p1704821039771309?thread_ts=1703131754.446429&cid=C069U77143Z) looks broken / permission blocked to me 🤔

Reply:
It's a private channel. Basically I was linking a discussion related to folder structure where someone was discussing if we consider E2E tests part of production code or something outside of it.

Right not it's outside and decoupled and it's not decided if it would be better to put playwright tests next to the production code in liferay-portal projects.

Reply:
For now let's assume we won't be accessing production code from tests. If everything settles in the future and we think it gives us any advantage, we may revisit it, I'd say.